### PR TITLE
Add optional task_definition variable, use it if non-empty

### PR DESCRIPTION
**Pull request by larstobi**
@phidah #33

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -212,7 +212,7 @@ resource "aws_ecs_service" "service" {
   ]
   name                               = var.name_prefix
   cluster                            = var.cluster_id
-  task_definition                    = aws_ecs_task_definition.task.arn
+  task_definition                    = var.task_definition != "" ? aws_ecs_task_definition.task.arn : var.task_definition
```
Comment (mhd999) [inline main.tf:215]: Should it be?
```
task_definition = var.task_definition != "" ? var.task_definition : aws_ecs_task_definition.task.arn 
```

Comment (larstobi) [reply main.tf:215]: You're right. Thanks!

Comment (larstobi) [inline main.tf:215]:
```suggestion
  task_definition                    = var.task_definition != "" ? var.task_definition : aws_ecs_task_definition.task.arn
```

**Review by mhd999 — APPROVED** [review]
LGTM

**Review by SadriG91 — APPROVED** [review]
LGTM

**Pull request merged**